packages/react-form/src/types.ts

```diff
@@ -9,6 +9,7 @@ export type DirtyStateComparator<Value> = (
 export interface Validator<Value, Context> {
   (value: Value, context: Context): ErrorValue;
 }
+export type {ErrorContent} from './validation';
```

Comment:
I don't think we need this?

The index file contains `export * from './types'; export * from './validation';` and you're already reexporting ErrorContent from validation.

Reply:
Oops. Good point! I'll update.